src/popup/locales/en.json

```diff
@@ -112,6 +115,18 @@
     "waitingToBeSent": "Waiting to be sent..."
   },
   "modals": {
+    "createAccount": {
+      "title": "Add account",
+      "titleMultisig": "Add multisig vault",
+      "titleAeternityAccount": "Add æternity account",
+      "msg": "Please select blockchain to add an account address to",
+      "msgMultisig": "Generate new multisig account address and add it to your wallet.",
+      "msgAeternity": "Generate new account address or import accounts from harware wallet.",
```

Comment:
1. It's a **hardware** wallet.
2. There are no support for hardware wallet right now. So the wording should be adjusted.